app/scripts/inpage.js

```diff
@@ -64,105 +60,9 @@ const inpageProvider = new MetamaskInpageProvider(metamaskStream)
-Set 'ethereum.autoRefreshOnNetworkChange' to 'false' to silence this warning: https://metamask.github.io/metamask-docs/API_Reference/Ethereum_Provider#ethereum.autorefreshonnetworkchange'
-`)
-    warnedOfAutoRefreshDeprecation = true
-  }
-  return new Promise((resolve, reject) => {
-    inpageProvider.sendAsync({ method: 'eth_requestAccounts', params: [force] }, (error, response) => {
-      if (error || response.error) {
-        reject(error || response.error)
-      } else {
-        resolve(response.result)
-      }
-    })
-  })
-}
-
-// give the dapps control of a refresh they can toggle this off on the window.ethereum
-// this will be default true so it does not break any old apps.
-inpageProvider.autoRefreshOnNetworkChange = true
-
-
-// publicConfig isn't populated until we get a message from background.
-// Using this getter will ensure the state is available
-const getPublicConfigWhenReady = async () => {
-  const store = inpageProvider.publicConfigStore
-  let state = store.getState()
-  // if state is missing, wait for first update
-  if (!state.networkVersion) {
-    state = await new Promise(resolve => store.once('update', resolve))
-    console.log('new state', state)
-  }
-  return state
-}
-
-// add metamask-specific convenience methods
-inpageProvider._metamask = new Proxy({
-  /**
-   * Synchronously determines if this domain is currently enabled, with a potential false negative if called to soon
-   *
-   * @returns {boolean} - returns true if this domain is currently enabled
-   */
-  isEnabled: function () {
-    const { isEnabled } = inpageProvider.publicConfigStore.getState()
-    return Boolean(isEnabled)
-  },
-
-  /**
-   * Asynchronously determines if this domain is currently enabled
-   *
-   * @returns {Promise<boolean>} - Promise resolving to true if this domain is currently enabled
-   */
-  isApproved: async function () {
-    const { isEnabled } = await getPublicConfigWhenReady()
-    return Boolean(isEnabled)
-  },
-
-  /**
-   * Determines if MetaMask is unlocked by the user
-   *
-   * @returns {Promise<boolean>} - Promise resolving to true if MetaMask is currently unlocked
-   */
-  isUnlocked: async function () {
-    const { isUnlocked } = await getPublicConfigWhenReady()
-    return Boolean(isUnlocked)
-  },
-
-  /**
-   * Registers a page as having initated onboarding. This facilitates MetaMask focusing the initiating tab after onboarding.
-   *
-   * @returns {Promise} - Promise resolving to undefined
-   */
-  registerOnboarding: async () => {
-    await promisify(onboardingStream.write({ type: 'registerOnboarding' }))
-  },
-}, {
-  get: function (obj, prop) {
-    !warned && console.warn('Heads up! ethereum._metamask exposes methods that have ' +
-    'not been standardized yet. This means that these methods may not be implemented ' +
-    'in other dapp browsers and may be removed from MetaMask in the future.')
-    warned = true
-    return obj[prop]
-  },
-})
+//
+// TODO:deprecate:2019-12-16
```

Comment:
@rekmarks  This date should be updated

Reply:
Fixed in 0950b7d (along with all other such examples)